infrastructure/helm-chart/charts/apps/charts/api/charts/api-admin/templates/service.yaml

```diff
@@ -3,9 +3,12 @@ kind: Service
 metadata:
   name: api-admin
   namespace: {{ .Values.global.namespace }}
+  labels:
+    type: spring
```

Comment:
Perhaps we should standardize these under the `core.airy.co` domain...

Maybe
```
core.airy.co/type: spring
```
is a better option?

Also we have already `    core.airy.co/managed: "true"`